docs/api/bluetooth/GattClient.md

```diff
@@ -16,6 +16,14 @@ As a result of the discovery process, the client can start interacting with the
 
 Mbed BLE abstracts read and write operations to offer a single API that can be used to read or write characteristic values. The application code does not have to handle the necessary fragmentation or reassembly process if the attribute value to be transported can't fit in a single data packet.
 
+#### ATT_MTU
+
+ATT_MTU is the maximum size of the attribute that can fit in an L2CAP packet. This is independent of the Data Length which controls the over-the-air packet payload size (which is dealt with in the GAP). An L2CAP packet will be fragmented over many such packets if required.
```

Comment:
I think we can improve the first sentence. What do you think of: 
`The GATT Maximum Transmission Unit (MTU) is the maximum size of a GATT packet exchanged between a client and a server.` Then mention L2CAP as a protocol that allows segmentation and reassembly, ...